src/components/BlogFeed.astro

```diff
@@ -67,6 +70,7 @@ type ArticleResult = {
             <img src={image.src} width={image.width} alt={media.data.alt} />
           </figure>
           <h4>{post.webTitle}</h4>
```

Comment:
Could we incorporate `rel="author"` on line 68 now we have the information?

Reply:
My understanding is that `rel="author"` should be used for a link that points to the author page specifically, rather than a post. In Guardian articles, this attribute is used to link to author's profile pages.

Luckily we can pass `show-tags=contributor` to the CAPI link and get back a useful response that looks like this: 

```typescript
tags: [
    {
      id: "profile/rasha-ardati",
      type: "contributor",
      webTitle: "Rasha Ardati",
      webUrl: "https://www.theguardian.com/profile/rasha-ardati",
      apiUrl: "https://content.guardianapis.com/profile/rasha-ardati",
      references: [ ],
      bio: "<p>Rasha is the Digital Co-ordinator in the Guardian Digital department<br></p>",
      firstName: "Rasha",
      lastName: "Ardati"
    }
],
```

Then, we can add an `a` tag with `rel="author"` that points to `tags.webUrl` (i.e. the author's profile page), which I think makes more sense.

Reply:
I've changed the code so that the byline now links to the author's profile page (using `rel="author"`). If the contributor tag isn't present then we fall back to a regular byline without an anchor tag.